frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -776,22 +778,43 @@ const ManagePolicyPage = ({
     }
   }
 
-  const renderPoliciesCount = (count?: number) => {
+  const renderPoliciesCountAndLastUpdated = (
+    count?: number,
+    policies?: IPolicyStats[]
+  ) => {
     // Hide count if fetching count || there are errors OR there are no policy results with no a search filter
     const isFetchingCount = !isAllTeamsSelected
       ? isFetchingTeamCountMergeInherited
       : isFetchingGlobalCount;
 
-    const hideCount =
+    const hide =
       isFetchingCount ||
       policiesErrors ||
       (!policyResults && searchQuery === "");
 
-    if (hideCount) {
+    if (hide) {
       return null;
     }
+    // Figure the time since the host counts were updated by finding first policy item with host_count_updated_at.
+    const updatedAt =
+      policies?.find((p) => !!p.host_count_updated_at)?.host_count_updated_at ||
+      "";
```

Comment:
Are these sorted?  Or is the expectation that all the policies will have the same `host_count_updated_at`, if they have one at all?

Reply:
I chose to just move the existing logic from [here](https://github.com/fleetdm/fleet/pull/25150/files#diff-f8327b6d753f9666bfa9f46dbaa95a9fc58384413757528a688db1a3ca262192L99-L114) to maintain existing behavior in this regard. Seems the assumption is that that's the case, which would make sense since I believe it's the same cron job that updates all policy host counts.